common/persistence/sql/sqlTaskPersistence.go

```diff
@@ -161,7 +175,7 @@ func (m *sqlTaskManager) LeaseTaskList(request *persistence.LeaseTaskListRequest
 		}
 
 		if result, err := tx.NamedExec(updateTaskListSQLQuery,
-			struct{
+			struct {
```

Comment:
Can you define these along with others in the definition section? Call it "updateTaskListRow" maybe.